openedx/core/djangoapps/appsembler/api/v1/api.py

```diff
@@ -33,6 +37,37 @@
 log = logging.getLogger(__name__)
 
 
+def enrollment_learners_context(course_id, identifiers):
+    """
+    Get emails (and learner language) from a list of learner identifiers.
+    :param course_id: The course ID.
+    :param identifiers: list of usernames/emails of students.
+    :return: iterator of tuples
+        (
+            user: User: The User object if found,
+            identifier: string: The dentifier as-is,
```

Comment:
typo. Should be "The identifier as-is."

Reply:
@johnbaldwin would you mind using the suggestion button 

![image](https://user-images.githubusercontent.com/645156/92612682-c4598600-f2c2-11ea-955e-3241dffedda3.png)

or tying it manually e.g.

    ```suggestion
    new line content
    ```

It makes a lot easier to pick changes as you intended it very quickly. Please check it out on: https://haacked.com/archive/2019/06/03/suggested-changes/